packages/vscode-integration/package.json

```diff
@@ -47,13 +47,12 @@
     "watch": "tsc -w"
   },
   "dependencies": {
-    "@eclipse-glsp/protocol": "2.1.0",
+    "@eclipse-glsp/protocol": "next",
```

Comment:
Isn't that risky to have a s a dependency when I rely on this package and there will be an update on the protocol?

Reply:
That's how we always do it during "nightly" development. The protocol version is fixed via yarn look anyways and can only be bumped with a "yarn upgrade:next".  (see also https://github.com/eclipse-glsp/glsp-theia-integration/blob/64a14998e22044a0ff3c2d220a3fa39f10509c26/packages/theia-integration/package.json#L47)